File: src/main/java/de/tectoast/emolga/utils/sql/managers/MusicGuildsManager.java
package de.tectoast.emolga.utils.sql.managers;

import de.tectoast.emolga.utils.sql.base.DataManager;
import de.tectoast.emolga.utils.sql.base.columns.LongColumn;

public class MusicGuildsManager extends DataManager {

    LongColumn GUILDID = new LongColumn("guildid", this);

    public MusicGuildsManager() {
        super("musicguilds");
        setColumns(GUILDID);
    }

    public void addGuild(long gid) {
        insert(gid);
    }
}

File: src/main/kotlin/de/tectoast/emolga/utils/sql/managers/MusicGuildsManager.kt
package de.tectoast.emolga.utils.sql.managers

import de.tectoast.emolga.utils.sql.base.DataManager
import de.tectoast.emolga.utils.sql.base.columns.LongColumn

object MusicGuildsManager : DataManager("musicguilds") {
    private val GUILDID = LongColumn("guildid", this)

    init {
        setColumns(GUILDID)
    }

    fun addGuild(gid: Long) {
        insert(gid)
    }
}